src/admin/containers/labels.js

```diff
@@ -33,18 +32,25 @@ const lines =
   , 'address3'
   , 'address4'
   , 'postcode'
-  , 'county'
+  , 'deliverer'
   ]
 
 
 const font_style = text =>
   ({ style: { fontSize: Math.min(12, 300 / text[0].length) + 'pt' } })
 
+
+const add_deliverer = arr =>
```

Comment:
I think this would be better as

`arr => over(lensIndex(arr.length - 1), defaultTo('POST'), arr)`